servicetalk-grpc-netty/src/main/java/io/servicetalk/grpc/netty/DefaultGrpcServerBuilder.java

```diff
@@ -93,8 +94,9 @@ final class DefaultGrpcServerBuilder implements GrpcServerBuilder, ServerBinder
 
     // Do not use this ctor directly, GrpcServers is the entry point for creating a new builder.
     DefaultGrpcServerBuilder(final Supplier<HttpServerBuilder> httpServerBuilderSupplier) {
-        this.httpServerBuilderSupplier = () -> httpServerBuilderSupplier.get()
-                .protocols(h2Default()).allowDropRequestTrailers(true);
+        this.httpServerBuilderSupplier = () ->
+                Objects.requireNonNull(httpServerBuilderSupplier.get(), "supplier result was null")
```

Comment:
consider static import

Reply:
consider including in the msg that we expected an HttpServerBuilder, something like:
`Supplier<HttpServerBuilder> result was null` or `Supplier result was null instead of a valid HttpServerBuilder instance`